kotlin/src/test/KotlinExample.kt

```diff
@@ -0,0 +1,109 @@
+
+import com.looker.rtl.ApiSettingsIniFile
+import com.looker.rtl.Transport
+import com.looker.rtl.UserSession
+import com.looker.sdk.*
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.apache.Apache
+import io.ktor.client.features.json.JacksonSerializer
+import io.ktor.client.features.json.JsonFeature
+import org.apache.http.conn.ssl.NoopHostnameVerifier
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy
+import org.apache.http.ssl.SSLContextBuilder
+import java.io.File
+import java.lang.Thread.sleep
+
+class KotlinExample {
+    val localConfig = "/Users/Looker/Documents/sdk_codegen/looker.ini"
+    val settings = ApiSettingsIniFile(localConfig, "Looker")
+
+    /**
+     * Note: This is only to work with self-signed certs... this should probably be removed before this is posted anywhere
```

Comment:
Probably worth changing this comment to specify that the pattern should not be used in production code instead. :D

Reply:
Changed